internal/resources/aws/global_accelerator.go

```diff
@@ -0,0 +1,348 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/infracost/infracost/internal/usage"
+	"github.com/shopspring/decimal"
+	"reflect"
+	"regexp"
+	"strings"
+)
+
+// GlobalAccelerator struct represents AWS Global Accelerator service
+//
+// Resource information: https://aws.amazon.com/global-accelerator
+// Pricing information: https://aws.amazon.com/global-accelerator/pricing/
+type GlobalAccelerator struct {
+	Name          string
+	IPAddressType string
+	Enabled       bool
+
+	MonthlyInboundDataTransferGB  *globalAcceleratorRegionDataTransferUsage `infracost_usage:"monthly_inbound_data_transfer_gb"`
+	MonthlyOutboundDataTransferGB *globalAcceleratorRegionDataTransferUsage `infracost_usage:"monthly_outbound_data_transfer_gb"`
+}
+
+type globalAcceleratorRegionDataTransferUsage struct {
```

Comment:
From the AWS docs, it looks like one side of the data transfer is determined by the region of the `aws_globalaccelerator_endpoint_group`.  So rather than create a usage param for every from/to permutation, we can create a new resource for the endpoint_group that have `monthly_inbound_data_transfer_gb` and `monthly_outbound_data_transfer_gb`, similar to the MonthlyDataTransfer... costs on the [cloud_front distribution resource](https://github.com/infracost/infracost/blob/master/internal/resources/aws/cloudfront_distribution.go).  So the usage file would look something like:
```
  aws_globalaccelerator_endpoint_group.my_endpoint:
    monthly_data_transfer_to_internet_gb:
      us: 12340000 # United States, Mexico, Canada
      europe: 22340000 # Europe, Israel
      south_africa: 32340000 # South Africa, Kenya, Middle East
      south_america: 42340000 # South America
      japan: 52340000 # Japan
      australia: 62340000 # Australia, New Zealand
      asia_pacific: 62340000 # Hong Kong, Philippines, Singapore, South Korea, Taiwan, Thailand
      india: 72340000 # India
    monthly_data_transfer_to_origin_gb:
      us: 1234 # United States, Mexico, Canada
      europe: 2234 # Europe, Israel
      south_africa: 3234 # South Africa, Kenya, Middle East
      south_america: 4234 # South America
      japan: 5234 # Japan
      australia: 6234 # Australia, New Zealand
      asia_pacific: 7234 # Hong Kong, Philippines, Singapore, South Korea, Taiwan, Thailand
      india: 8234 # India
      ```